src/CharacterStateSerializer.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+
+import CharacterState from './CharacterState';
+
+export default class CharacterStateSerializer {
+    characterToPositionLookUpTable: {[string]: number};
+    constructor() {
+        this.characterToPositionLookUpTable = {
```

Comment:
I think it would be really good if we could implement this as a function, the inverse of `getAlphabetCoord()` that we test in a similar manner to the test for `getAlphabetCoord()` where we test at '0', 'a', 'A', 'z', and 'Z'.